src/blocks/remote-data-container/hooks/usePaginationVariables.ts

```diff
@@ -88,6 +89,7 @@ export function usePaginationVariables( {
 		supportsCursorPagination || supportsPagePagination || supportsOffsetPagination;
 	const totalItems = paginationData?.totalItems;
 	const totalPages = totalItems && perPage ? Math.ceil( totalItems / perPage ) : undefined;
+	const hasNextPage = paginationData?.hasNextPage;
```

Comment:
can you update the `supports*Pagination` variables above to take into account the statement we made in the schema? "either total_items or `has_next_page` must be defined"

Reply:
Something like this? 
```js
	const supportsPagination =
		supportsCursorPagination ||
		supportsPagePagination ||
		supportsOffsetPagination ||
		totalItems ||
		hasNextPage;
```

Reply:
@chriszarate I've added this here [`b5e7856` (#391)](https://github.com/Automattic/remote-data-blocks/pull/391/commits/b5e7856adf60918d891d852e1ec69f060a5d967c), but let me know if you were thinking of something else!